codice/ddf
# DDF-2100: Adds registry-common

#### What does this PR do?

Adds the updated version of registry-common.
The updates include adding test coverage for RegistryObjectMetacardType, deleting an unused file (RegistryQueryDelegate), adding and modifying registry constants, and adding transient attributes to the RegistryObjectMetacardType.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@mcalcote @gordocanchola @clockard @oscaritoro 
@coyotesqrl (Hero)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@shaundmorris

#### How should this be tested?
Run `mvn clean install` and verify the build runs successfully. 
No manual steps are required since this is just commons.

#### What are the relevant tickets?
DDF-2100
